src/client/components/app/LineDetailsDisplay/index.tsx

```diff
@@ -44,8 +73,12 @@ export const LineDetailsDisplay = (props: LineDetailsProps) => {
   const StandardMetadata = (metadataProps: any) => (
     <>
       <Header>
-        <BorrowerName ellipsis>{borrower}</BorrowerName>
-        's Line Of Credit
+        <RouterLink to={`/portfolio/${borrower}`} key={borrower} selected={false}>
```

Comment:
can u verify that when u click these that the analytics events go off

Reply:
Yes, with internal links analytics go off, it seems like market line cards don't trigger analytics though. But that was not part of this PR

![ANALYTICS](https://user-images.githubusercontent.com/69639595/208242026-05b1924c-3381-42dc-8647-2221db839016.png)